specification/liftrqumulo/resource-manager/Qumulo.Storage/preview/2022-06-27-preview/Qumulo.Storage.json

```diff
@@ -0,0 +1,753 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Qumulo.Storage",
+    "version": "2022-06-27-preview",
```

Comment:
>2022-06-27-preview", [](http://example.com/codeflow?start=16&length=20)

why such a backdated release?

Reply:
We are in private preview currently and have public preview scheduled in mid December. we want to currently support this old release as well until we are stable with new release. we plan to remove this release at later point.